### DISCUSSION TITLE
請問大佬們這個觸發器怎麽使用的

### DISCUSSION THREAD

**Discussion by StealthEmissarI** [AutomaApp/automa#783] [General]
我拖拽出來，鬆手就消失了，5555555
![QQ图片20220911221216](https://user-images.githubusercontent.com/61230741/189532170-22073a0c-0387-4a1c-85c0-aa41bffebd53.jpg)

**Comment by Kholid060** [comment]
You don't need a trigger if it's a package. And if you enable the `Set as a package` option, you need to define the input and the output of the package https://docs.automa.site/guide/packages.html#set-package-as-block
